test/ResourceXml.test.js

```diff
@@ -0,0 +1,1721 @@
+/*
+ * ResourceXML.test.js - test the rule that checks XML matching
+ *
+ * Copyright © 2024 JEDLSoft
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ResourceString } from 'ilib-tools-common';
+import { IntermediateRepresentation, Result, SourceFile } from 'ilib-lint-common';
+
+import ResourceXML from "../src/rules/ResourceXML.js";
+
+const sourceFile = new SourceFile("a/b/c.xliff", {});
+
+describe("testResourceXML", () => {
+    test("ResourceXML", () => {
+        expect.assertions(1);
+
+        const rule = new ResourceXML();
+        expect(rule).toBeTruthy();
```

Comment:
What was your intention here?
Asking, because, this test will pass for any object, even an empty one. To be more specific you can use `toBeInstanceOf()` matcher.